instagram/src/LogIn.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import LoginSubmit from './Components/SearchBar/LoginSubmit';
+
+
+class LogIn extends Component {
+
+  constructor() {
+    super();
+    this.state = {
+        login: [],
+      username: [],
+      password: [],
+      uInput:'',
+      pInput:'',
+
+    } 
+  }
+
+  componentDidMount() {
+    this.setState({login: LoginSubmit}) 
+
+    }
+
+  uInputHandler = event => {
+    const value = event.target.value;
+    this.setState (
+      {uInput:value}
+    );
+
+    }
+
+    pInputHandler = event => {
+        const value = event.target.value;
+        this.setState (
+          {pInput:value}
+        );
+
+
+        }
+
+    submitHandler = event => {
+      event.preventDefault();
+        this.state.username.unshift(this.state.uInput);
+        this.state.password.unshift(this.state.pInput);
+        console.log('Congratulations! You are Now Signed Up.');
+    }
+
+    loginHandler = event => {
+        event.preventDefault();
```

Comment:
This method is super creative! I love the extra addition of the sign up and login. However, it is not rendering correctly right now. Try to implement just the login set to localStorage for now :)